src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java

```diff
@@ -2,69 +2,69 @@
 
 import com.softserveinc.ita.dkrutenko.pageobjects.softserve.ContactUsPage;
 import com.softserveinc.ita.dkrutenko.pageobjects.softserve.MainPage;
+import com.softserveinc.ita.dkrutenko.pageobjects.softserve.usermodel.User;
+import com.softserveinc.ita.dkrutenko.pageobjects.softserve.usermodel.UserRepository;
 import com.softserveinc.ita.dkrutenko.utils.runners.TestRunner;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
+import static com.codeborne.selenide.Selenide.open;
 
 public class SoftServeTests extends TestRunner {
```

Comment:
if you opening browsers in tests, not needed extending of testrunner

Reply:
but in testrunner i have my profile options and urls. Should i put them into test class?

Reply:
Atm removed for softserve tests only